client/components/MainPage.js

```diff
@@ -1,18 +1,10 @@
+/* eslint-disable radix */
```

Comment:
I would prefer telling the code to explicitly use base 10 rather than disabling this linter warning.

Reply:
commit a1bb21c resolves this